zxlive/graphscene.py

```diff
@@ -291,7 +293,9 @@ def mouseReleaseEvent(self, e: QGraphicsSceneMouseEvent) -> None:
 
     def add_vertex(self, e: QGraphicsSceneMouseEvent) -> None:
         p = e.scenePos()
-        self.vertex_added.emit(*pos_from_view(p.x(), p.y()))
+        # edges under current mouse position
+        edges: list[EItem] = [e for e in self.items(p, deviceTransform=QTransform()) if isinstance(e,EItem)]
```

Comment:
I figured out how to do check for edge intersection by creating a rectangle around the mouse position. This can be based on the grid size. I will push that change.

Reply:
Pushed now!